collectors/0/samza.py

```diff
@@ -101,5 +132,20 @@ def print_metric(metric_type, metric_name, ts, value, **tags):
         print ("%s.%s %d %s %s" % (metric_type.replace('org.apache.', ''), metric_name, ts, value, tags))
 
 
+def print_consumer_lag(ts, value, **tags):
+
+    if (unicode(str(value), 'utf-8').isnumeric()):
+        if tags:
+          tags = " " + " ".join("%s=%s" % (sanitize(name), v)
+                                for name, v in tags.iteritems())
+        else:
+            tags = ""
+        print ("%s %d %s %s" % (SAMZA_CONSUMER_LAG_METRIC_NAME, ts, value, tags))
```

Comment:
We might want to report this using the standard metric name as opposed to relabeling it. This way it's easier to find from a samza perspective and we don't need any special logic here.